packages/docs/sdk/react-sdk/api/useCreateChannel.mdx

```diff
@@ -14,15 +14,15 @@ import { useCreateChannel } from '@towns-protocol/react-sdk'
 ```ts
 function useCreateChannel(
   spaceId: string,
-  config?: ActionConfig<Space['createChannel']>,
+  config?: ActionConfig<Space["createChannel"]>,
 ): {
     data: string | undefined;
     error: Error | undefined;
     isPending: boolean;
     isSuccess: boolean;
     isError: boolean;
     createChannel: (channelName: string, signer: Signer, opts?: {
-        topic?: string | undefined;
```

Comment:
this is a different signature... is this also a docs update?

Reply:
`topic?: string` is the same as `topic?: string | undefined`. Typescript Eslint added a new default rule that removes the explicit undefined from the latter.

Reply:
>  is this also a docs update?

I had to update the api-extractor package, which changed a bit some stuff (i.e the single quote vs double quote in docs types)

I also had to simplify `useObservable` return type in react-sdk, since by some reason it was hitting an error where the api-extractor wasn't able to extract the type, failing to generate docs